content/log/explore.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is actualy in private beta. If you'd like to apply to participate in the private beta, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview 
+
+The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.
+
+{{< img src="log/explore/explore_view_with_comments.png" alt="Explore view with comments" >}}
+
+On this view you have:
```

Comment:
A bit "raw"

I would change it to:

In this view you can:

- Interact with the Time range
- Display logs into lists
- Setup columns in log lists ( I would actually merge it in the previous section)
- Use facets to help you in various investigations
- Enter search queries

Reply:
👍